docs/tutorial/force_calibration.rst

```diff
@@ -104,3 +118,94 @@ We can plot the calibration by calling::
     calibration.plot()
 
 .. image:: force_calibration_fit.png
+
+Hydrodynamically correct model
+------------------------------
+
+While the idealized Lorentzian model discussed in the previous section is often sufficiently accurate, there are scenarios where more detailed models are necessary.
+
+The idealized model is based on the assumption that the drag force is only proportional to the bead's velocity.
+This assumption is realistic when the bead moves at a constant speed with respect to the fluid.
+When oscillating a bead however, this frictional force becomes frequency dependent.
+
+The following equation accounts for a frequency dependent drag coefficient :cite:`tolic2006calibration`:
+
+.. math::
+
+    P_{hydro}(f) = \frac{D Re(\gamma / \gamma_0)}{\pi^2 \left(\left(f_{c,0} + f Im(\gamma/\gamma_0) - f^2/f_{m, 0}\right)^2 + \left(f Re(\gamma / \gamma_0)\right)^2\right)}
+
+where the corner frequency is given by:
+
+.. math::
+
+    f_{c, 0} = \frac{\kappa}{2 \pi \gamma_0}
+
+and :math:`f_{m, 0}` parameterizes the time it takes for friction to dissipate the kinetic energy of the bead:
+
+.. math::
+
+    f_{m, 0} = \frac{\gamma_0}{2 \pi m}
+
+with :math:`m` the mass of the bead.
+Finally, :math:`\gamma` corresponds to the frequency dependent drag coefficient.
+Assuming a measurement deep in bulk, then :math:`\gamma` = :math:`\gamma_{stokes}` which is given by:
+
+.. math::
+
+    \gamma_{stokes} = \gamma_0 \left(1 + (1 - i)\sqrt{\frac{f}{f_{\nu}}} - \frac{2}{9}\frac{f}{f_{\nu}} i\right)
+
+Where :math:`f_{\nu}` is the frequency at which the penetration depth in the liquid of the bead’s linear harmonic motion equals the radius of the bead.
+The penetration depth characterizes the exponential decrease of the fluid’s velocity field as a function of the distance from a bead that is forced to do linear harmonic motion with frequency :math:`f` :cite:`berg2006power`:.
+It is given by :math:`4 \nu/(\pi d^2)` with :math:`\nu` the kinematic viscosity.
+
+This approximation is reasonable, when the bead is far from the surface.
+When approaching the surface, the drag experienced by the bead depends on the distance between the center of the bead and the nearest surface of the flow cell.
+An approximate expression for the frequency dependent drag coefficient is then given by :cite:`tolic2006calibration`:
+
+.. math::
+
+    \gamma(f, R/l) = \frac{\gamma_{stokes}(f)}{1 - \frac{9}{16}\frac{R}{l}\left(1 - \left((1 - i)/3\right)\sqrt{\frac{f}{f_{\nu}}} + \frac{2}{9}\frac{f}{f_{\nu}}i - \frac{4}{3}(1 - e^{-(1-i)(2l-R)/\delta})\right)}
+
+Where :math:`\delta = R \sqrt{\frac{f_{\nu}}{f}}` represents the aforementioned penetration depth and :math:`R` corresponds to the bead radius, :math:`l` to the distance from the bead center to the nearest surface.
```

Comment:
....depth,  :math:`R` corresponds to the bead radius _and_ :math:`l` to the.....